src/mappings/helpers/bridge.ts

```diff
@@ -50,9 +51,18 @@ export function getRetryableTicketId(
   let fields: ByteArray[] = [];
 
   // Get the L2 chain id based on the L1 network
-  // 0x066EED = 421613 (Arbitrum Goerli)
-  // 0xA4B1 = 42161 (Arbitrum One)
-  let l2ChainIdHex = addresses.network === "mainnet" ? "0xA4B1" : "0x066EED";
+  let l2ChainIdHex = "";
+  if (addresses.network === "mainnet") {
+    l2ChainIdHex = "0xA4B1"; // 0xA4B1 = 42161 (Arbitrum One)
+  } else if (addresses.network === "goerli") {
+    l2ChainIdHex = "0x066EED"; // 0x066EED = 421613 (Arbitrum Goerli)
+  } else if (addresses.network === "sepolia") {
+    l2ChainIdHex = "0x066EEE"; // 0x066EEE = 421614 (Arbitrum Sepolia)
+  } else {
+    log.critical('Unsupported network: {}', [addresses.network]);
```

Comment:
This would break all deployments except mainnet, goerli and sepolia, since `log.critical` causes the subgraph to crash. If we want to give a warning or error message, please change it to `log.error` or `log.warning`

Reply:
I see, my intention was to be loud so next time we wouldn't forget to update this but we can't do that since we do deploy this to other networks. I'll update to a warning.

Reply:
Error would probably be also a decent choice if you want to make it extra obvious that something isn't expected, but given the context warning makes sense too